src/cocotb/handle.py

```diff
@@ -296,6 +303,28 @@ def _discover_all(self) -> None:
 
         self._discovered = True
 
+    def _get(
+        self, key: str, discovery_method: GPIDiscovery = GPIDiscovery.AUTO
+    ) -> Optional[simulator.gpi_sim_hdl]:
+        """Query the simulator for an object with the specified *key*.
+
+        Like Pythons native dict get-function, this returns None if the object
+        is not found instead of raising an AttributeError.
```

Comment:
```suggestion
        Like Python's native dictionaty ``get``-function, this returns ``None`` if the object
        is not found instead of raising an :exc:`AttributeError`.
```